src/libs/dynamodb-lib.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  DynamoDBClient,
+  PutItemCommand,
+  GetItemCommand,
+  GetItemCommandInput,
+} from "@aws-sdk/client-dynamodb";
+import {
+  marshall,
+  unmarshall,
+  NativeAttributeValue,
+} from "@aws-sdk/util-dynamodb";
+
+const client = new DynamoDBClient({ region: process.env.region });
+
+export async function putItem({
+  tableName,
+  item,
+}: {
+  tableName: string;
+  item: { [key: string]: NativeAttributeValue };
+}) {
+  const params = {
+    TableName: tableName,
+    Item: marshall(item, {
+      removeUndefinedValues: true,
+    }),
+  };
+
+  try {
+    if (item && item.id) console.log(`Putting item with id: ${item.id}:`);
```

Comment:
we dont have id on the items anymore so we either need to change what we are checking for here, or remove this line

Reply:
I updated it 
try {
    if (item && item.recordId)
      console.log(`Putting item with id: ${item.recordId}:`);

Reply:
try {
    if (item && item.recordId)
      console.log(`Putting item with recordId: ${item.recordId}:`);